sklearn/linear_model/_base.py

```diff
@@ -101,38 +101,57 @@ def _deprecate_normalize(normalize, default, estimator_name):
     else:
         _normalize = normalize
 
+    pipeline_msg = (
+        "If you wish to scale the data, use Pipeline with a StandardScaler "
+        "in a preprocessing stage. To reproduce the previous behavior:\n\n"
+        "from sklearn.pipeline import make_pipeline\n\n"
+        "model = make_pipeline(StandardScaler(with_mean=False), "
+        f"{estimator_name}())\n\n"
+        "If you wish to pass a sample_weight parameter, you need to pass it "
+        "as a fit parameter to each step of the pipeline as follows:\n\n"
+        "kwargs = {s[0] + '__sample_weight': sample_weight for s "
+        "in model.steps}\n"
+        "model.fit(X, y, **kwargs)\n\n"
+    )
+
+    if estimator_name == 'Ridge' or estimator_name == 'RidgeClassifier':
+        alpha_msg = 'Set parameter alpha to: original_alpha * n_samples. '
+    elif 'Lasso' in estimator_name:
+        alpha_msg = (
+            'Set parameter alpha to: original_alpha * np.sqrt(n_samples). '
+        )
+    elif 'ElasticNet' in estimator_name:
+        alpha_msg = (
+            'Set parameter alpha to original_alpha * np.sqrt(n_samples) if'
+            'l1_ratio is 1 and original_alpha * n_samples if l1_ratio is 0. '
```

Comment:
What about `0 < l1_ratio < 1`?

Reply:
I guess we don't have a closed form formula for this case.

Reply:
```suggestion
            'Set parameter alpha to original_alpha * np.sqrt(n_samples) if'
            'l1_ratio is 1, and to original_alpha * n_samples if l1_ratio is 0. '
            'For other values of l1_ratio, there is no analytic formula available.'
```

Reply:
This is the only remaining review comment. I think a little more info as suggested would be helpful. Otherwise, I anticipate new issues for exactly that.
After that, I'll intend to merge.